### PR TITLE
fix(components): Enumeration header style

### PR DESCRIPTION
**Pull request by jare-talend**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
maxime/gwandal review

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://251.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://251.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://251.talend.surge.sh)

File: packages/components/src/Enumeration/Header/Header.scss [hunk 1]
```diff
@@ -7,14 +7,12 @@ $tc-enumeration-header-height: 28px !default;
...
-	text-transform: uppercase;
 	align-items: center;
 	height: $tc-enumeration-header-height;
 
 	span {
-		font-size: 12px;
-		font-weight: 700;
-		letter-spacing: 0.6px;
+		font-size: 1.5rem;
+		font-weight: normal;
```
Comment (charlesnguyen) [inline packages/components/src/Enumeration/Header/Header.scss]: Not necessary since it is by default.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://251.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://251.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://251.talend.surge.sh)

**Pull request merged**